pandas/io/excel/_odswriter.py

```diff
@@ -110,6 +113,14 @@ def _write_cells(
         )
         from odf.text import P
 
+        if notes is not None:
+            raise NotImplementedError(
+                """
+                Notes are not supported by the odswriter engine,
+                see https://github.com/eea/odfpy
+                """
```

Comment:
This should be a regular string, `"Notes are not..."`, not a triple quoted string.